### PR TITLE
If greeting sample

### PR DESCRIPTION
**Pull request by LevapVeeskela**

### CODE REVIEW

File: Practice/Practice/Program.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            const string EveningText = "Good evening, guys!";
+            const string NightText = "Good night, guys!";
+
+            var today = DateTime.Now.TimeOfDay;
+            
+            if (today.Hours > 9 && today.Hours < 12)
+            {
+                Console.WriteLine(MorningText);
+            }
+            else if (today.Hours > 12 && today.Hours < 15)
```
Comment (VadzimPapko) [inline Practice/Practice/Program.cs]: According the task requirements - today.Hours > 12 should be `today.Hours >= 12`

File: Practice/Practice/Program.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                Console.WriteLine(DayText);
+            }
+            else if (today.Hours > 15 && today.Hours < 22)
```
Comment (VadzimPapko) [inline Practice/Practice/Program.cs]: According the task requirements - today.Hours > 15 should be `today.Hours >= 15`

File: Practice/Practice/Program.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                Console.WriteLine(EveningText);
+            }
+            else if (today.Hours > 22 && today.Hours < 9)
```
Comment (VadzimPapko) [inline Practice/Practice/Program.cs]: According the task requirements - today.Hours > 22 should be `today.Hours >= 22`

today.Hours < 9 should be `today.Hours <= 9`

File: Practice/Practice/Program.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                Console.WriteLine(NightText);
+            }
+
+            var stringTime = $"Right now time: {today.Hours}:{today.Minutes}:{today.Seconds}";
+            Console.WriteLine(stringTime);
+            Console.ReadKey();
+
```
Comment (VadzimPapko) [inline Practice/Practice/Program.cs]: this empty line could be deleted

File: Practice/Practice/Program.cs (hunk 3 above)
Comment (LevapVeeskela) [reply Practice/Practice/Program.cs]: I agree with you, I will fix now

**Pull request merged**